### PR TITLE
Plugin and URL configs for Open edX

### PR DESCRIPTION
**Pull request by OmarIthawi**
Make URLs discoverable by Open edX using the "Hawthorn" plugins system.

### CODE REVIEW

File: mocks/openedx/core/djangoapps/plugins/constants.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+"""
+Mocks for openedx.core.djangoapps.plugins.constants.
+
+This mostly stupid mocks just to make the plugin app works.
```
Comment (johnbaldwin) [inline mocks/openedx/core/djangoapps/plugins/constants.py]: Maybe remove "stupid"

File: mocks/openedx/core/djangoapps/plugins/constants.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+"""
+
+from mock import Mock
+
+# Just a lazy way to avoid complex mocking.
```
Comment (johnbaldwin) [inline mocks/openedx/core/djangoapps/plugins/constants.py]: Another way to look at it is that this provides a minimal mocking to reduce test maintenance costs

Comment (OmarIthawi) [reply mocks/openedx/core/djangoapps/plugins/constants.py]: It's always better to see the filled half of the glass.

File: mocks/openedx/core/djangoapps/plugins/constants.py (hunk 1 above)
Comment (OmarIthawi) [reply mocks/openedx/core/djangoapps/plugins/constants.py]: I like it. Thanks!

**Pull request merged**